### PR TITLE
8291924: jpackage: l10n for Windows context menu label

### PR DESCRIPTION
**Pull request by akashche**
This change adds `ContextMenuCommandLabel` l10n property for file association context menu label. It is a follow-up to [this PR comment](https://github.com/openjdk/jdk/pull/9224#issuecomment-1169286082).

Note, non-EN l10n values were filled using auto-translator and may need to be corrected.

To check the contents of the label, registry query is added to `PackageTest::addHelloAppFileAssociationsVerifier`.

Testing:

 - [x] ran `FileAssociationsTest` with `install` enabled

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8291924](https://bugs.openjdk.org/browse/JDK-8291924): jpackage: l10n for Windows context menu label

### Reviewers
 * [Naoto Sato](https://openjdk.org/census#naoto) (@naotoj - **Reviewer**) ⚠️ Review applies to [227ad5d4](https://git.openjdk.org/jdk/pull/9753/files/227ad5d45be3161eb94df909e0ce5e0b738f34f8)
 * [Alexey Semenyuk](https://openjdk.org/census#asemenyuk) (@alexeysemenyukoracle - **Reviewer**)
 * [Alexander Matveev](https://openjdk.org/census#almatvee) (@sashamatveev - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/9753/head:pull/9753` \
`$ git checkout pull/9753`

Update a local copy of the PR: \
`$ git checkout pull/9753` \
`$ git pull https://git.openjdk.org/jdk pull/9753/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 9753`

View PR using the GUI difftool: \
`$ git pr show -t 9753`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/9753.diff">https://git.openjdk.org/jdk/pull/9753.diff</a>

</details>

### CODE REVIEW

File: test/jdk/tools/jpackage/helpers/jdk/jpackage/test/WindowsHelper.java [hunk 1]
```diff
@@ -356,7 +356,7 @@ private void verifyFileAssociations(boolean exists, String suffix,
         private final String name;
     }
 
-    private static String queryRegistryValue(String keyPath, String valueName) {
+    public static String queryRegistryValue(String keyPath, String valueName) {
```
Comment (alexeysemenyukoracle) [inline test/jdk/tools/jpackage/helpers/jdk/jpackage/test/WindowsHelper.java:359]: It should be sufficient to make it package private, not public.

**Review by naotoj — COMMENTED** [review]
As to the l10n files, no need to provide those. Providing the English resource is just fine. L10n will pick them up in the later translation process.

**Comment by akashche** [comment]
I've updated the patch, making `queryRegistryValue` package private, removing non-EN l10n entries and adding minor enhancements to label value check.

File: test/jdk/tools/jpackage/helpers/jdk/jpackage/test/WindowsHelper.java (hunk 1 above)
Comment (akashche) [reply test/jdk/tools/jpackage/helpers/jdk/jpackage/test/WindowsHelper.java]: Changed the modifier to package private.

**Review by naotoj — APPROVED** [review]
Looks good for the translation file change.

**Comment by alexeysemenyukoracle** [comment]
I think there was a miscommunication about l10n files.
All "MsiInstallerStrings_\*.wxl" files need to be updated, but no need to provide a localized version of English string(s). 
I.e. add `<String Id="ContextMenuCommandLabel">Open with [ProductName]</String>` to all "MsiInstallerStrings_*.wxl" files, not only to "MsiInstallerStrings_en.wxl".

**Comment by naotoj** [comment]
Missing entries in localized files are automatically detected by the translation process. So modifying only the English file should be fine.

**Comment by akashche** [comment]
Thanks for the l10n files review! As I understand, I need a second jpackage-specific review before integrating the change. I can change non-EN `.wxl` files as needed - please confirm if they need to be changed.

**Comment by alexeysemenyukoracle** [comment]
@naotoj , I suspect missing entries in l10n files will cause code depending on them to fail when executed in non-English locales until the next l10n drop corrects missing entries.

**Comment by naotoj** [comment]
Thanks, @alexeysemenyukoracle , I was under the impression of falling back to the English resource if l10n was not available. If it is not falling back to English,  please bring those non-English resources back.

**Comment by akashche** [comment]
Thanks for clarification! I've updated the patch adding English entries to all `.wxl` files.

**Review by alexeysemenyukoracle — APPROVED** [review]

**Review by sashamatveev — APPROVED** [review]

**Comment by akashche** [comment]
I've verified that there is no fallback to EN for `.wxl` files (more details in [this comment](https://github.com/openjdk/jdk/pull/9780#issuecomment-1210685780)), so leaving EN entries that were added to all `.wxl` files.

**Pull request closed without merging**